molecule/default/molecule.yml

```diff
@@ -1,118 +1,24 @@
 ---
 dependency:
   name: galaxy
+
 driver:
   name: docker
 
 platforms:
-  # deb
```

Comment:
Must not remove the existing tests.  We could fail to detect regressions in the other installers with this change.